go/hyperkit.go

```diff
@@ -440,6 +455,22 @@ func (h *HyperKit) buildArgs(cmdline string) {
 	log.Debugf("hyperkit: CmdLine: %#v", h.CmdLine)
 }
 
+// openTty opens the tty files for reading, and returns it.
+func (h *HyperKit) openTty() *os.File {
```

Comment:
TTY is an abbreviation so either `openTTY` or `opentty` throughout would be preferable.